roles/foreman_installer_devel_scenario/templates/katello-devel-answers.yaml

```
@@ -1,11 +1,11 @@
 ---
 certs:
-  group: vagrant
+  group: {{ foreman_installer_devel_scenario_group }}
```

Thinking more: should we even have a default for the group or should it default to the user instead:
```suggestion
  group: {{ foreman_installer_devel_scenario_group | default(foreman_installer_devel_scenario_user) }}
```

---

I like the suggestion, because it allows the user to pass one less variable in the case where the user and group should match (probably most of the time) but still allows for the full customization in case it is required.